Project02Eclipse/src/com/hemebiotech/analytics/ReadSymptomDataFromFile.java

```diff
@@ -31,7 +36,7 @@ public ReadSymptomDataFromFile (String filepath) {
 				BufferedReader reader = new BufferedReader (new FileReader(filepath));
 				String line = reader.readLine();
 				
-				while (line != null) {
+				while (line != null) { // we browse the file line by line then we add the content of each line in the result array

```

Comment:
The comments are only accepted on functions/properties definitions. If you need to comment a code line, it means that it's not correctly coded. You can remove this one.

Reply:
Ok, i imagine it's also valid for the other files.
It's done.